src/setup.py

```diff
@@ -598,6 +745,121 @@ def main():
                                  "Creating backup of the original configuration")
     REPLACE_WHITELIST_CMD = CMD(COPY_CMD.format(source=RECOMMENDED_WHITELIST, target=DEFAULT_PLUGIN_CONFIGURATION_DIR), "Replacing whitelist configuration")
 
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+        description='Script for custom installation process for collectd AWS CloudWatch plugin'
+    )
+    parser.add_argument(
+        '-I', '--non_interactive', required=False,
+        help='Non interactive mode',
+        default=False, action='store_true'
+    )
+    parser.add_argument(
+        '-H', '--host_name', required=False,
+        help='Manual override for EC2 Instance ID and Host information propagated by collectd',
+        metavar='HOST_NAME', default=None
+    )
+    parser.add_argument(
+        '-r', '--region', required=False,
+        help='Manual override for region used to publish metrics',
+        metavar='REGION', default=None
+    )
+    parser.add_argument(
+        '-n', '--proxy_name', required=False,
+        help='Proxy server name',
+        metavar='NAME', default=None
+    )
+    parser.add_argument(
+        '-p', '--proxy_port', required=False,
+        help='Proxy server port',
+        metavar='PORT', default=None
+    )
+    parser.add_argument(
+        '-e', '--enable_high_resolution_metrics', required=False,
+        help='Enable high resolution metrics',
+        metavar='ENABLE_HIGH_RESOLUTION_METRICS', default=None
```

Comment:
Could you change the default value to false?

if it is a string, line 486 will always be true.
    line 486: if self.enable_high_resolution_metrics:

Reply:
Done.